Darkly/DarklyConstants.m

```diff
@@ -5,7 +5,7 @@
 #import "DarklyConstants.h"
 
 NSString * const kClientVersion = @"0.4.2";
-NSString * const kBaseUrl = @"https://app.launchdarkly.com/msdk/eval/users";
+NSString * const kBaseUrl = @"https://mobile.launchdarkly.com";
```

Comment:
this won't work, will it? It changes the URL not just for events but for fetching flags.